README.md

```diff
@@ -112,4 +112,8 @@ the bug report template. If you think something isn't working, don't hesitate to
 
 ## institutional and funding partners
 
-![CZI logo](https://chanzuckerberg.com/wp-content/themes/czi/img/logo.svg)
+<picture>
+  <source media="(prefers-color-scheme: dark)" srcset="https://chanzuckerberg.com/wp-content/themes/czi/img/logo-white.svg">
+  <source media="(prefers-color-scheme: light)" srcset="https://chanzuckerberg.com/wp-content/themes/czi/img/logo.svg">
```

Comment:
Does this line is required? The source looks identical to the default one.

Reply:
Hi @Czaki, good catch.  Since the `src` tag is set to the logo for the light theme, we only need the `srcset` tag for the dark theme.  I have updated the pull request.